src/components/FetchCars.jsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable comma-spacing */
+/* eslint-disable quotes */
+import { Link } from "react-router-dom";
+import {
+  FaChevronLeft,
+  FaChevronRight,
+  FaFacebookF,
+  FaLinkedinIn,
+} from "react-icons/fa";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import splitSentence from "../helpers/SplitSentence";
+
+const FetchCars = () => {
+  const [cars, setCars] = useState([]);
+  const [currentGroupIndex, setCurrentGroupIndex] = useState(0);
+  const groupSize = 3;
+  const totalGroups = Math.ceil(cars.length / groupSize);
+
+  const handleNextGroup = () => {
+    setCurrentGroupIndex((prevIndex) => (prevIndex + 1) % totalGroups);
+  };
+
+  const handlePrevGroup = () => {
+    setCurrentGroupIndex(
+      (prevIndex) => (prevIndex - 1 + totalGroups) % totalGroups
+    );
+  };
+
+  const displayByThree = cars.slice(
+    currentGroupIndex * groupSize,
+    currentGroupIndex * groupSize + groupSize
+  );
+
+  useEffect(() => {
+    const fetchCarsData = async () => {
+      const res = await axios.get("https://jsonplaceholder.typicode.com/posts");
+      setCars(res.data);
+    };
+    fetchCarsData();
+  }, []);
```

Comment:
## Optional
As the API endpoints of our backend is completed, you can change the URL to fetch data from our API instead. You can apply this in this PR or in a different PR later, its up to your preference.

Reply:
Thank you for your review.  I appretiate your suggestions. But I could not find backend api ready.  Will tell me on this please?